### DISCUSSION TITLE
When the title of an article contains Chinese, the anchor cannot locate the title

### DISCUSSION THREAD

**Discussion by wqdygkd** [outline/outline#4158] [Self Hosting] [answered]
outline : v0.66.0

Here is the test: https://docs.wqdy.top/share/605e6a5d-2e6f-49df-8bd4-df10586c8b94#h-this-title-contain-chinese-中文

**Comment by tommoor** [comment, marked as answer]
Fix will be in the next release. (https://github.com/outline/outline/commit/61154ba618ab8e1598f491216622fffd426b178d)

**Reply by wqdygkd** [reply]
👍
